src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1400,8 +1404,6 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node, format
                 (output_layout.format == format::b_fs_yx_fsv16) &&
                 !needs_onednn_bfyx_to_blocked(format::bfyx, output_layout.format, input_layout, conv))
```

Comment:
What does this condition mean? Can't we remove `first_conv` and `needs_onednn_bfyx_to_blocked` here?

Reply:
|    8 | 133                          | Convolution    |  2090 | [7]      | [1, 1, 128, 100]  | b_fs_yx_f32    | U8       | [1, 1, 128, 100]  | b_fs_yx_f32    | U8       | ocl:ref:any__u8                  |

Reply:
Modified